model/src/test/java/com/ge/predix/acs/model/AttributeTest.java

```diff
@@ -0,0 +1,27 @@
+package com.ge.predix.acs.model;
+
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class AttributeTest {
```

Comment:
As discussed, it's debatable how much value POJO-related unit tests provide. Maybe we should create such tests once we've identified gaps via code coverage?